src/mame/heathzenith/README.md

```diff
@@ -0,0 +1,53 @@
+# Heath Company / Zenith Data Systems
```

Comment:
I don't know that we've done anything like this before.  I'm not totally against it, but my initial feeling is that this is more like something that would either be in source comments or the wiki: https://wiki.mamedev.org/index.php/System-Specific_Setup_and_Information

@cuavas @galibert Do you guys have any feelings about this?

Reply:
It doesn’t belong here.  It’s a combination of trivia that belongs on Wikipedia, usage information that belongs on the MAMEdev wiki, and notes that belong in source files.